app/schema/album.js

```diff
@@ -9,9 +10,16 @@ exports.schema = buildSchema(`
   },
   type Query {
       albums: [album]
-  }
+  },
+  type Mutation{
+    deleteAlbum(id:Int): String
+    }	  
 `);
 
 exports.root = {
-  albums: () => getResources('/albums')
+  albums: () => getResources('/albums'),
+  deleteAlbum: async (args, req) => {
+    const deleteAlbumPurchased = await Album.deteleAlbumPurchased({ userId: req.user.id, albumId: args.id });
+    return deleteAlbumPurchased === 1 ? 'Album  is deleted' : 'Album not found';
```

Comment:
Error handling seems tricky, try to implement `formatError` for handling of errors